ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
 = Package and deploy applications with Open Liberty
 
 You can choose among different options for packaging, containerizing, and deploying applications with Open Liberty.
-However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize server packages with Docker, and deploy to the cloud with Kubernetes or OpenShift.
+However, the simplest workflow for developing your cloud-native applications is to build applications with Maven or Gradle, containerize with Docker, and deploy to the cloud with Kubernetes or OpenShift.
 
-== Package
-Before you containerize and deploy your application, package it with the Open Liberty runtime.
-There are different tools that you can use to package your application.
-Using Maven or Gradle to build and package your application is one of the easiest ways to get your application ready for cloud deployment.
-Maven and Gradle both work well as part of continuous integration and continuous deployment (CI/CD) pipelines.
-They start with the Open Liberty kernel and download specific feature versions that are listed as dependencies.
-
-=== Maven
+== Maven and Gradle build tools
 link:https://maven.apache.org/[Maven] is a build automation tool that makes managing your builds and dependencies easier.
 With Maven, you use a Project Object Model (POM) file written in XML to define information about your project, and Maven downloads and installs all dependencies for you.
 The link:https://github.com/OpenLiberty/ci.maven[Open Liberty Maven plug-in] helps manage your Open Liberty server and applications.
```

Comment:
The phrase "helps manage your Open Libert server and application" could be read as meaning our plugin is for system management, rather than the plugin is there to help develop and package applications at the development phase.